fearless/Modules/Staking/SelectValidatorsFlow/ValidatorSearch/ValidatorSearchViewController.swift

```diff
@@ -69,22 +81,17 @@ final class ValidatorSearchViewController: UIViewController, ViewHolder, Importa
     }
 
     private func setupNavigationBar() {
-        let rightBarButtonItem = UIBarButtonItem(
-            title: "",
-            style: .plain,
-            target: self,
-            action: #selector(tapDoneButton)
-        )
-
         let attributes: [NSAttributedString.Key: Any] = [
             .foregroundColor: R.color.colorWhite()!,
             .font: UIFont.p0Paragraph
         ]
 
-        rightBarButtonItem.setTitleTextAttributes(attributes, for: .normal)
-        rightBarButtonItem.setTitleTextAttributes(attributes, for: .highlighted)
+        doneButton.setTitleTextAttributes(attributes, for: .normal)
```

Comment:
I believe that for validator filters and your validators we use `setupDefaultTitleStyle` of UIBarButtonItem to setup style. Should we apply it also here?

Reply:
Fixed